ChangeLog

```diff
@@ -1,19 +1,63 @@
 CHANGES
 =======
 
+v1.0.10
```

Comment:
curious why this is coming in in your PR @hrshdhgd ?

Reply:
I noticed that too! Whenever I do `python setup.py install` when first setting up my virtual environment, this one and the 'AUTHORS' file get generated.

Reply:
A product of the PBR loader -- pretty useful